### ISSUE TITLE
[NEW] Add an option on the onboarding so user can choose to send or not anonymous usage data

### ISSUE THREAD

**Issue by localguru** [RocketChat/Rocket.Chat.Android#1757]
## Description

The automatic activation of "Analytics Tracking" is extremely questionable from a data protection point of view, even if the data is anonymized, because connection data is still transmitted and this is legally personal data.

In my opinion, the better option would be to politely ask the user after the installation whether he/she would like to support the quality and further development of the client with the activation.

Personally, I support tracking, but I know users who take a "hidden" tracking very negatively.

## Devices and Versions

Your Rocket.Chat.Android version: 2.6.1

**Comment by TwizzyDizzy** [comment]
True, this is not GDPR-compliant without conscent (agreement  - active!, not passive - to some kind of data policy detailing what happens to the data and who gets it - maybe even more, I'm no particular expert there).

**Comment by rafaelks** [comment]
@localguru I agree with this, let's find a way to ask the user to support sending anonymous information once the app starts the first time. And yes, all data is always anonymous, there's no sensitive/private information there.

**Comment by rafaelks** [comment]
Fabric [is ready for GDPR](https://docs.fabric.io/android/fabric/data-privacy.html#fabric-is-gdpr-ready), but we're studying other solutions, to actually ask the user before we start sending information.

**Comment by RotBolt** [comment]
On onbaoarding of screen wouldn't be choosing activity sufficient for opting to send the data ?
If so could I proceed with this ?
